client/src/networking/networking.js

```diff
@@ -49,32 +51,45 @@ export default ({ children }) => {
         initializeEventHandlers(io);
     }
 
+    
```

Comment:
Delete extra space -- Maybe we should figure out linting at some point (prettier?)